test/cppsuite/tests/poc.cxx

```diff
@@ -0,0 +1,22 @@
+#include <iostream>
+extern "C" {
+    #include "wiredtiger.h"
+}
+
+int main(int argc, char *argv[]) {
+    WT_CONNECTION *conn;
+    int ret = 0;
+    // Setup basic test directory.
+    std::string default_dir = "WT_TEST";
```

Comment:
The `default_dir` and `mkdir_cmd` strings can be marked as const.

Reply:
4a87eda